desc/compute/_profiles.py

```diff
@@ -216,6 +216,26 @@ def _Te_rr(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="<ne>_rho",
+    label="\\bar{n}_e",
+    units="m^{-3}",
+    units_long="1 / cubic meters",
+    description="Line-averaged electron density",
```

Comment:
Can you change to 
```python
label="\\bar{n}_e = \\int n_e d\\rho"  # Simple radial average〈nₑ〉ᵨ = ∫ nₑ dρ 
```
to emphasize your average does not include radial integration weight like $\Vert e_{\rho} \Vert$ and then average.